Wox/ViewModel/ResultViewModel.cs

```diff
@@ -31,13 +33,12 @@ public ImageSource Image
                     catch (Exception e)
                     {
                         Log.Exception($"|ResultViewModel.Image|IcoPath is empty and exception when calling Icon() for result <{Result.Title}> of plugin <{Result.PluginDirectory}>", e);
-                        return ImageLoader.Load(Result.IcoPath);
```

Comment:
This is a good pick up, thanks for this- of course it will throw exception since IcoPath is null in the first place. Cheers